ubuntu-advantage-client/azure_cleanup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+"""Clean up all running Azure resources and resource groups."""
+
+# Copyright 2020 Canonical Ltd.
+# Lucas Moura <lucas.moura@canonical.com>
+
+from pycloudlib.azure.util import get_client
+from azure.mgmt.compute import ComputeManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+
+import argparse
+
+
+CI_DEFAULT_TAG = "uaclient"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-t", "--tag", dest="tag", action="store", default=CI_DEFAULT_TAG,
+        help=("Tag used to filter cloud resources for deletion. "
+              "Default: {}".format(CI_DEFAULT_TAG))
+    )
+    parser.add_argument(
```

Comment:
it'd be good at some point if we could leverage a credential file vs args so that they weren't leaked in run logs and such.

Reply:
Got it, I will add that option